src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1261,22 +1267,6 @@
   {:pre [(kitchensink/datetime? time)]}
   (jdbc/delete! :reports ["producer_timestamp < ?" (to-timestamp time)]))
 
-(defn delete-old-unassociated-catalogs!
-  [time]
-  {:pre [(kitchensink/datetime? time)]}
-  (when @store-catalogs-historically?
-    (let [time (to-timestamp time)]
-      (jdbc/do-prepared
-       (str "DELETE FROM catalogs oc "
-            "       WHERE oc.producer_timestamp < ? "
-            "       AND NOT EXISTS (SELECT 1 FROM catalogs c "
-            "                       JOIN reports r ON c.catalog_uuid = r.catalog_uuid "
```

Comment:
it seems like by eliminating consideration of reports we must be changing behavior when report-ttl is < 3 days. I don't understand why this function isn't needed anymore.